examples/cpp/pyperf/README.md

```diff
@@ -8,12 +8,26 @@ It works with Python 2.7, and 3.6 through 3.9.
 
 ```
 PyPerf [-p|--pid PID]
-       [-d|--duration DURATION_MS]
-       [-F|--frequency FREQUENCY_HZ]
+       [-c|--sample-rate TICKS]
+       [-F|--frequency HZ]
+       [-d|--duration SECONDS]
+       [--update-interval SECONDS]
+       [--symbols-map-size SIZE]
+       [--events-buffer-pages PAGES]
+       [--kernel-stacks-map-size SIZE]
+       [--user-stacks-pages PAGES]
        [-v|--verbose VERBOSITY]
        [-o|--output FILENAME]
+       [--fs-offset OFFSET]
+       [--stack-offset OFFSET]
 ```
 
+`--frequency` and `--sample-rate` are mutually exclusive, and analogous to `sample_freq` and `sample_period` from `perf_event_open(2)`:
+
+> A "sampling" event is one that generates an overflow notification every N events, where N is given by `sample_period`.   `sample_freq` can be used if you wish to use frequency rather than period. In this case, you set the freq flag. The  kernel will adjust the sampling period to try and achieve the desired rate.  The rate of adjustment is a timer tick.
+
+Thus, frequency is the number of desired samples per second, whereas rate is number of timer ticks between samples.
```

Comment:
I don't think we should have this here, because the `sample-rate` flag should be removed anyway as it's not useful to us.